projects/elife.yaml

```diff
@@ -1975,6 +1975,17 @@ elife-xpub:
         demo:
             s3:
                 "{instance}-elife-xpub":
+        staging:
+            s3:
+                "{instance}-elife-xpub":
+            rds:
+                engine: postgres # or 'MySQL'
+                version: '10.4'
+                type: db.t2.small
```

Comment:
could use `db.t2.micro` (1 GB of RAM instead of 2 GB)?

Reply:
As long as we document that staging is not prepared for load tests.

Reply:
Definitely, a separate temporary environment perfectly replicating `prod` should be used for that.